### PR TITLE
Setup and merge stripe payments with dev

### PR DESCRIPTION
**Pull request by alextheprogrammer21**

### CODE REVIEW

File: client/src/components/Stripe.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    });
+  };
+  render() {
+    return (
+      <StripeCheckout
+        token={this.onToken}
+        amount={this.data.amount}
+        email={this.data.email}
+        currency="USD"
+        stripeKey="pk_test_51IBWOAIGBdXhz29TWRY6MjXouhgfvf3wtJMx3b3hwEozhBQ1so2qxP5WpFLdBwMX0IpIdJsC2LJ0EmKwkBAYHqEF005V3RV8A9" //Publishable Key
```
Comment (jessdesr) [inline client/src/components/Stripe.js]: this should still be put in ENV file to match how it would function if you were using a real production key

**Review by jessdesr — COMMENTED** [review]
Is there anything here that is saving the stripe customer to the account that is making the payment, or was that going to be implemented in a later ticket?

**Comment by alextheprogrammer21** [comment]
> Is there anything here that is saving the stripe customer to the account that is making the payment, or was that going to be implemented in a later ticket?

Hi Jesse, it does actually have the functionality to save the customer to the payment. The stripe component that's currently being used allows saving of user details based on email by checking the remember me option. The saving is persistent and will stay saved

File: client/src/components/Stripe.js (hunk 1 above)
Comment (alextheprogrammer21) [reply client/src/components/Stripe.js]: Updated

File: server/routes/stripe.js [hunk 2]
```diff
@@ -1,15 +1,40 @@
...
+    res.json({
+      message: "Payment Successful",
+      success: true,
+    });
+  } catch (error) {
+    res.json({
+      message: "Payment Failed",
+      success: false,
+    });
+  }
```
Comment (jusa798) [inline server/routes/stripe.js:37]: nice comments! Makes it easier to follow :)

**Review by jusa798 — APPROVED** [review]

**Pull request merged**